app/workers/notify_vacancy_on_slack_worker.rb

```diff
@@ -0,0 +1,16 @@
+require "slack_notifier"
+
+class NotifyVacancyOnSlackWorker
+  include Sidekiq::Worker
+  include Rails.application.routes.url_helpers
+
+  def perform(vacancy_id)
+    vacancy = Vacancy.find(vacancy_id)
+    message = %Q(
```

Comment:
I think we can have the message here.. After, if needed, we extract it. Is it ok for you @lastk ?